Map.h

```diff
@@ -0,0 +1,31 @@
+
+#include <string>
+#include <vector>
+#include <list>
+#include <utility>
+#include <sstream>
+#include <cctype>
+#include <algorithm>
+#include <iostream>
+#include "FileManager.h"
+
+/*
+ * Header File For Map Class
+ */
+class Map
+{
+public:
+	bool Export(std::string distinctWord, int value, std::string fileName, bool eof);
+	bool map(std::string fileName, std::string value, bool eof);
+	Map(FileManager *fm, std::string intermediateDir);
```

Comment:
Do we want FileManager being inserted as an argument? If we are including it within the class as an included header, we can just create a new instance within the code at a later time